synth.py

```diff
@@ -26,31 +26,32 @@
 gapic = gcp.GAPICMicrogenerator()
 # keep v1beta1 for the time being:
 library = gapic.typescript_library(
-  'secretmanager',
-  generator_args={
-    "grpc-service-config": "google/cloud/secrets/v1beta1/secretmanager_grpc_service_config.json",
-    "package-name": "@google-cloud/secret-manager",
-  },
-  proto_path='/google/cloud/secrets/v1beta1',
-  version='v1beta1')
-s.copy(library, excludes=['package.json', 'README.md', 'src/index.ts'])
+    'secretmanager',
+    generator_args={
+        "grpc-service-config": "google/cloud/secrets/v1beta1/secretmanager_grpc_service_config.json",
+        "package-name": "@google-cloud/secret-manager",
+    },
+    proto_path='/google/cloud/secrets/v1beta1',
+    version='v1beta1')
+s.copy(library, excludes=['package.json', 'README.md'])
 # run the gapic generator (for all versions except v1beta1):
 versions = ['v1']
 library = 'secretmanager'
 for version in versions:
-  library = gapic.typescript_library(
-    library,
-    generator_args={
-      "grpc-service-config": f"google/cloud/{library}/{version}/secretmanager_grpc_service_config.json",
-      "package-name": "@google-cloud/secret-manager",
-    },
-    proto_path=f'/google/cloud/{library}/{version}',
-    version=version)
-s.copy(library, excludes=['package.json', 'README.md', 'src/index.ts'])
+    library = gapic.typescript_library(
+        library,
+        generator_args={
+            "grpc-service-config": f"google/cloud/{library}/{version}/secretmanager_grpc_service_config.json",
+            "package-name": "@google-cloud/secret-manager",
+        },
+        proto_path=f'/google/cloud/{library}/{version}',
+        version=version)
+s.copy(library, excludes=['package.json', 'README.md'])
 
```

Comment:
These two blocks seem to be similar, can we combine them together as one loop? Thanks

Reply:
"grpc-service-config" address is different. I think that's the reason why divide into two loop.

Reply:
I see, LGTM then